File: selector/src/main/java/com/luck/picture/lib/interfaces/OnCustomLoadingListener.java
package com.luck.picture.lib.interfaces;

import android.app.Dialog;
import android.content.Context;

/**
 * @author：luck
 * @date：2022/8/12 7:19 下午
 * @describe：OnCustomLoadingListener
 */
public interface OnCustomLoadingListener {
    /**
     * create loading dialog
     *
     * @param context
     */
    Dialog create(Context context);
}

File: selector/src/main/java/com/luck/picture/lib/interfaces/OnCustomLoadingListener.kt
package com.luck.picture.lib.interfaces

import android.app.Dialog
import android.content.Context

/**
 * @author：luck
 * @date：2022/8/12 7:19 下午
 * @describe：OnCustomLoadingListener
 */
interface OnCustomLoadingListener {
    /**
     * create loading dialog
     *
     * @param context
     */
    fun create(context: Context): Dialog
}